src/traces/parcoords/lines.js

```diff
@@ -176,7 +176,7 @@ function setAttributes(attributes, sampleCount, points) {
 function emptyAttributes(regl) {
     var attributes = {};
     for(var i = 0; i < 16; i++) {
-        attributes['p' + i.toString(16)] = regl.buffer({usage: 'dynamic', type: 'float', data: null});
+        attributes['p' + i.toString(16)] = regl.buffer({usage: 'dynamic', type: 'float', data: new Uint8Array()});
```

Comment:
Shouldn't this be `new Uint8Array(0)` just like you did in:

https://github.com/dy/regl-scatter2d/commit/c1183ef38f11df62f2301d0ca2f1316121a90604#diff-f0cb24e401db59c6759d5cb577e58cebR403